test/bigtcp/test.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
Please add the license/copyright header. I know not all bash scripts have it, but let's try to add it for new files.